sig-cloud-provider/README.md

```diff
@@ -23,12 +23,20 @@ The [charter](CHARTER.md) defines the scope and governance of the Cloud Provider
 ### Chairs
 The Chairs of the SIG run operations and processes governing the SIG.
 
+* Andrew Sy Kim (**[@andrewsykim](https://github.com/andrewsykim)**), Google
+* Nick Turner (**[@nckturner](https://github.com/nckturner)**), Amazon
+
+### Technical Leads
```

Comment:
Don't feel strongly but this might be better in a separate PR

Reply:
Happy to do it either way, I'll leave it for now out of laziness but if @justaugustus feels like it should be separate then I'll split it out.